lib/delayed_job/delayed_worker.rb

```diff
@@ -56,6 +58,16 @@ def get_initialized_delayed_worker(config, logger)
     Delayed::Worker.max_run_time = config.get(:jobs, :global, :timeout_in_seconds) + 1
     Delayed::Worker.sleep_delay = config.get(:jobs, :global, :worker_sleep_delay_in_seconds)
     Delayed::Worker.logger = logger
+    if ::Sequel::Model.db.database_type == :mysql
+      read_ahead = config.get(:jobs, :read_ahead) || Delayed::Worker::DEFAULT_READ_AHEAD
+      # lock for update is not configurable for mysql
+      read_ahead = Delayed::Worker::DEFAULT_READ_AHEAD if read_ahead <= 0
+    else
+      # read_ahead 0 = lock for update (default for postgres)
+      read_ahead = config.get(:jobs, :read_ahead) || 0
+      read_ahead = 0 if read_ahead < 0
```

Comment:
Would it make sense to extract the 0 constant to a variable?
E.g. `Delayed::Worker::DEFAULT_READ_AHEAD_POSTGRES`?